README.md

```diff
@@ -18,74 +16,84 @@ This library is part of [ACPaaS UI][acpaas-ui].
-| notifications  | TODO                                     | [Documentation](./packages/notifications/README.md)       |
-| pagination     | TODO                                     | [Documentation](./packages/pagination/README.md)          |
-| progress-bar   | TODO                                     | [Documentation](./packages/progress-bar/README.md)        |
-| selectable-list| TODO                                     | [Documentation](./packages/selectable-list/README.md)     |
-| table          | TODO                                     | [Documentation](./packages/table/README.md)               |
-| utils          | TODO                                     | [Documentation](./packages/utils/README.md)               |
+| agenda         | Calendar to display events               | [Documentation](./packages/agenda/README.md)              |
+| analytics      | Google Analytics integration             | [Documentation](./packages/analytics/README.md)           |
+| avatar         | User avatar icon                         | [Documentation](./packages/avatar/README.md)              |
+| calendar       | Calendar to select dates                 | [Documentation](./packages/calendar/README.md)            |
+| code-snippet   | Snippet of source code                   | [Documentation](./packages/code-snippet/README.md)        |
+| context        | Set meta tags for SEO                    | [Documentation](./packages/context/README.md)             |
+| flyout         | Reveal additional content                | [Documentation](./packages/flyout/README.md)              |
+| forms          | Assorted form fields                     | [Documentation](./packages/forms/README.md)               |
+| layout         | Assorted page layout components          | [Documentation](./packages/layout/README.md)              |
+| localstorage   | Interface with localstorage              | [Documentation](./packages/localstorage/README.md)        |
+| logo           | Logo icon                                | [Documentation](./packages/logo/README.md)                |
+| map            | Leaflet-based map                        | [Documentation](./packages/map/README.md)                 |
+| notifications  | Show user notifications                  | [Documentation](./packages/notifications/README.md)       |
+| pagination     | Pagination control                       | [Documentation](./packages/pagination/README.md)          |
+| progress-bar   | Progress bar control                     | [Documentation](./packages/progress-bar/README.md)        |
+| selectable-list| List control with selectable items       | [Documentation](./packages/selectable-list/README.md)     |
+| table          | Interactive table                        | [Documentation](./packages/table/README.md)               |
+| utils          | Assorted helpers                         | [Documentation](./packages/utils/README.md)               |
```

Comment:
Maybe we should also list the submodules, like auto-complete?

Reply:
Should be added here when documenting the forms module (links to titles inside the forms readme).

Reply:
True. Good call.